### PR TITLE
Fix name of variable

### PR DESCRIPTION
**Pull request by pothitos**
##### SUMMARY
The variable ansible_ssh_user had to be changed to match the context.

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

### CODE REVIEW

File: docs/docsite/rst/user_guide/playbooks_variables.rst [hunk 1]
```diff
@@ -1074,7 +1074,7 @@ If you want to override the remote user globally (even over inventory) you can u
 the ``lola`` value is still ignored, but ``ansible_user=maria`` takes precedence over all other places where ``ansible_user`` (or ``remote_user``) might be set.
 
 A connection-specific version of a variable takes precedence over more generic
-versions.  For example, ``ansible_ssh_user`` specified as a group_var would have
+versions.  For example, ``ansible_user`` specified as a group_var would have
```
Comment (bcoca) [inline docs/docsite/rst/user_guide/playbooks_variables.rst:1077]: this change is incorrect, we are specifically pointing out that `ansible_ssh_user`, which is specific to the ssh connection plugin overrides `ansible_user` ... having it override itself makes no sense

**Review by bcoca — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/user_guide/playbooks_variables.rst (hunk 1 above)
Comment (pothitos) [reply docs/docsite/rst/user_guide/playbooks_variables.rst:1077]: Hello Brian!

I thought that the meaning of my change is that
  + `ansible_user` specified as a group_var would have a higher precedence than `ansible_user` specified as a host_var

and not
  + `ansible_user` would have a higher precedence than `ansible_user`.

It seems that I miss something. Anyway, as the deprecated `ansible_ssh_user` is not mentioned anywhere else, if this pull request is closed, someone may open an issue to further explain it.

Comment (bcoca) [reply docs/docsite/rst/user_guide/playbooks_variables.rst:1077]: no, your change makes it incorrect, the hostvars have precedence over groupvars of the same name

also, ansible_ssh_user is NOT deprecated, it was moved from 'global config' to 'connection plugin config'

Comment (pothitos) [reply docs/docsite/rst/user_guide/playbooks_variables.rst:1077]: You're right, Brian, my change is wrong. Anyway, the latest version is more clear than the one I edited.

However, please note that `ansible_ssh_user` is stated as deprecated in [windows_winrm.rst](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/user_guide/windows_winrm.rst#inventory-options) in the Note under the "Inventory Options" section.

**Pull request closed without merging**